README.md

```diff
@@ -1,28 +1,279 @@
+
+| Preset      | Description | 
+| ----------- | ----------- | 
+| BS2G (Large Scale)      | BS2G crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm           |
+| BS2G (Small Scale)    | BS2G crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm            |
+| SDA    | SDA crosslinker, K/S/T/Y/Nterm to any amino acid       |
+| BS3 (Large Scale)    | BS3 crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm. Also for DSS.    |
+| BS3 (Small Scale)    |  BS3 crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm. Also for DSS. |
+| DSSO (Large Scale)    | DSSO crosslinker,  K/S/T/Y/Nterm to  K/S/T/Y/Nterm  with cleavable stub fragment identification      |
+| DSSO (Small Scale)    | DSSO crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm with cleavable stub fragment        |
+| EDC    | EDC crosslinker, K/S/T/Y, Nterm to E/D/Cterm. No modifications defined.       |
+| DSBU (Large Scale)    | DSBU crosslinker, K/S/T/Y/Nterm with cleavable stun fragment   |
+| DSBU (Small Scale)    | DSBU crosslinker, K/S/T/Y/Nterm with cleavable stun fragment        |
+| NonCovalent    | Include noncovalent identification in search    |
+| Linear Search    | perform linear peptide search ONLY (overrides all other options)        |
+
+
+##### Tolerance 
+Set the MS1 and MS2 search tolerances. If you are working with high-resolution orbitrap (120K MS1 and 60k MS2) data that has been previously recalibrated with msfragger or a linear search, we suggest very tight tolerances such as 3ppm MS1 and 5ppm MS2. Non-recalibrated data is usually searched with looser tolerances such as 6ppm MS1 and 10ppm MS2. Notice that xiSEARCH does not perform recalibration. Thus, some information on MS1 and MS2 error from quality control runs or linear proteomic searches of the same samples is necessary to set sensible tolerances.
+
+
+#### Enzyme
+Select an enzyme or multiple enzyme used to digest the sample. 
+
+| Preset                   | Description                             | 
+|--------------------------|-----------------------------------------| 
+| Trypsin                  |                                         |
+| Trypsin/P                | trypsin not restricted by proline       |
+| V8                       |                                         |
+| Lys-C                    |                                         |
+| Lys-C/P                  |                                         |
+| trypsin/P + V8           |                                         |
+| proteinaseK              |                                         |
+| proteinaseK & trypsin\P  |                                         |
+| Chymotrypsin             |                                         |
```

Comment:
in the GUI, these options are full of typos and inconsistencies. Chymotrypsin is sometimes chemotrypsin, multiple digestions are sometimes with a slash and other times with a +... consider fixing

Reply:
I will go through the BasicConfigEntries.conf file and have a look